README.md

```diff
@@ -59,16 +55,7 @@ We support several serializers (or you can bring your own):
 > `AllowSynchronousIO` to `true` as per [this announcement](https://github.com/aspnet/Announcements/issues/342);
 > which isn't recommended.*
 
-You can get the latest pre-release packages from the [MyGet feed](src/NuGet.config),
-where you may want to explicitly pull a certain version using `-v`.
-```
-> dotnet add package GraphQL.SystemTextJson -v 3.0.0-preview-1593
-```
-
-[MyGet feed](https://www.myget.org/F/graphql-dotnet/api/v3/index.json) is the primary source of packages
-where you can find all preview versions built from the `master` branch. Periodically (usually once every
-few months) the latest preview version is published to NuGet manually. This is due to fairly frequent
-changes. Publication of each preview version to NuGet would create only unnecessary noise.
+You can get all preview versions from [GitHub registry](https://github.com/orgs/graphql-dotnet/packages?repo_name=graphql-dotnet).
```

Comment:
```suggestion
You can get all preview versions from [GitHub Packages](https://github.com/orgs/graphql-dotnet/packages?repo_name=graphql-dotnet).
Note that GitHub requires authentication to consume the feed.  See [here](https://docs.github.com/en/free-pro-team@latest/packages/publishing-and-managing-packages/about-github-packages#authenticating-to-github-packages).
```

Reply:
It would be nice to have a little more documentation about how to actually use a package from GitHub Packages, with a sample NuGet.config file and an outline of how to create a PAT and configure a Windows computer to authenticate and use the feed.  Perhaps inside the docs rather than here in the readme, with a link to the docs from here.

Reply:
You understand this more than I do.

GitHub Packages or GitHub registry ? First I wrote Packages but then switched to registry. How is it correct?

Reply:
GitHub Packages is the name of GitHub’s registry

Reply:
Since all their documentation uses “GitHub Packages” (as it relates to this), and the packages are listed under the title of “Packages” on our repo homepage, I’d use “GitHub Packages”

Reply:
Maybe I can add some docs....